### ISSUE TITLE
: Explainable AI and interpretability of AI solutions: Strategic overview challenges and caveats.

### ISSUE THREAD

**Issue by mekriti** [pydatadelhi/talks#103]
- Abstract: With the exponential increase in computing power available, several AI algorithms that were mere papers written decades ago have become implementable. For a data scientist, it is very tempting to use the most sophisticated algorithm available for every kind of problem. But given that its applicability has moved beyond academia and out into the business world, are numbers alone sufficient? Putting context to AI, or XAI (explainable AI) takes the black box out of AI to enhance human computer interaction. This talk shall revolve around the overview, challenges, drivers and caveats of the XAI paradigm, and an intuitive demo of translating inner workings of an ML algorithm into human understandable formats to achieve more business buy-ins.

- Pre-requisites for the talk

- Time: 1 to 1.5 hour

- Link to slides

- Will you be doing hands-on demo as well? Yes

- Link to ipython notebook (if any): 
https://www.kaggle.com/kritidoneria/explainable-ai-xai-example-20190615
https://www.kaggle.com/kritidoneria/explainable-ai-eli5-lime-and-shap
(Will be changed most probably)

- About yourself
Currently, Kriti works closely to design data driven business centric decision support systems for Fortune 500 companies. She is an active and published researcher in the domain on emerging technologies and algorithms (Cloud computing, AI/Ml, AR/VR etc).She has also been one of the highest rated competitive coders in the country.
Kriti holds an MBA (Analytics and marketing) from IIT Kanpur and B.Tech in computer science.

- Are you comfortable if the talk is recorded and uploaded to PyData Delhi's YouTube channel ? Yes

- Any query ?
I would like to bring a co-speaker along with me.
[ Mr. Mohd. Saqib Atiq 
https://github.com/verdecken]

**Comment by sanketverma1704** [comment]
Please review @shagunsodhani.

**Comment by shagunsodhani** [comment]
Thanks for the talk proposal @mekriti It looks interesting. If you are planning to make it a ~1.5 hour talk, maybe you could split it into two parts. The two parts can be delivered on the same day - with a 10 min break :)

**Comment by mekriti** [comment]
Hi,
Thanks for the suggestion.It sounds doable to me.
I will try to keep it shorter but laying the groundwork is essential before I move to the hands-on demo.An audience demographic beforehand would be great.
It can be split because anyway people would need time to set up their systems (if any) in between..

**Comment by sanketverma1704** [comment]
Hi @mekriti.
Would you like to present this in our upcoming meetup?

**Comment by mekriti** [comment]
Hi team,
Tentative date?
Thanks and regards,
Kriti

**Comment by sanketverma1704** [comment]
19th October.

**Comment by mekriti** [comment]
Hi Team,
I confirm my availability.
As mentioned earlier,I'd like to bring a co-speaker with me[cc'd].
Let us know how to go about it.
Thanks and regards,
Kriti

**Comment by sanketverma1704** [comment]
Great. I'll let you know once I'll update the agenda. Thanks!

**Comment by mekriti** [comment]
Thanks.

**Comment by sanketverma1704** [comment]
Thanks for the talk, closing this now.
